legion/engine/physics/data/component_flags.hpp

```diff
@@ -7,8 +7,10 @@ namespace legion::physics
     {
         pc_add_first_box = 0ull,
         pc_add_first_sphere,
+        pc_add_first_convex,
```

Comment:
I know this sucks to do, and honestly idm if you leave it like this. but the propper way to do this that prevents namespace clogging with enums is using `enum class`/`enum struct`.
and then for flag usage you can overload the bitwise operators of the enum class, and also overload the implicit cast operator to `size_type`

Reply:
I'm gonna leave this one in, mostly because the other enums added later on in the next subfeatures are also using plain enums